flow.yml

```diff
@@ -30,12 +29,11 @@ executors:
     needs: [clip_encoder]
   - name: rerank
     uses: jinahub+docker://CLIPTorchEncoder/latest-gpu
-    host: 'demo-cas.jina.ai'
+    host: 'api.clip.jina.ai'
     port: 2096
     uses_requests:
       '/': rank
-    tls: true
-    external: true
+    external: false
```

Comment:
this has to remain `true`

Reply:
If the default is `true` to use an external executor, the flow cannot work without providing the correct token. It's a bit of noising.

Reply:
okey I see, why was it `true` by default before?

Reply:
before, we use `demo-cas.jina.ai` as default. Thus, `external` is true.